Doc/library/math.rst

```diff
@@ -298,9 +298,12 @@ Number-theoretic and representation functions
 
 .. function:: trunc(x)
 
-   Return the :class:`~numbers.Real` value *x* truncated to an
-   :class:`~numbers.Integral` (usually an integer). Delegates to
-   :meth:`x.__trunc__() <object.__trunc__>`.
+   Given a :class:`~numbers.Real` value *x*, return *x* with the fractional part
+   removed, leaving the integer part.  This rounds toward 0: ``trunc()`` is
+   equivalent to :func:`floor` for positive *x*, and equivalent to :func:`ceil` for
+   negative *x*. If *x* is not a float, ``trunc()`` delegates to
+   :meth:`x.__trunc__ <object.__trunc__>`, which should return an
+   :class:`~numbers.Integral` value if properly defined.
```

Comment:
It was my intention that this sentence be a copy of the corresponding sentences in the ceil and floor entries.
 The latter reads "If *x* is not a float, delegates to ``x.__floor__()``, which should return an
:class:`~numbers.Integral` value."

```suggestion
   negative *x*. If *x* is not a float, delegates to
   :meth:`x.__trunc__ <object.__trunc__>`, which should return an
   :class:`~numbers.Integral` value.
```
I left the markup change, though, since it makes the method name link to the Data model entry.  I think that this is useful, but if we do it here, we should do it for ceil and floor.  Otherwise, make a separate issue/PR for making the code a link.